psi4/driver/schema_wrapper.py

```diff
@@ -586,6 +586,18 @@ def run_json_qcschema(json_data, clean, json_serialization, keep_wfn=False):
             )):
                 json_data["extras"]["qcvars"][k] = _serial_translation(v, json=json_serialization)
 
+    # Add in handling of matrix arguments which need to be obtained by a
+    # a function call.
+
+    if "psi4:arrays" in json_data["extras"] :
```

Comment:
do you need separate options for each array? I'm not keen on empty arrays in the input schema as a signal to save amps. what about something like below?
```
if json_data["extras"].get("psi4:save_tamps", False):
    json_data["extras"]["psi4:arrays"]["tIjAb"] = wfn.get_amplitudes()["tIjAb"].to_array().tolist()
    json_data["extras"]["psi4:arrays"]["tIA"] = wfn.get_amplitudes()["tIA"].to_array().tolist()
    json_data["extras"]["psi4:arrays"]["Da"] = wfn.Da().to_array().tolist()
```